tests/test_inference.py

```diff
@@ -5495,12 +5495,14 @@ def add(x, y):
                 add(*nums)
                 print(**kwargs)
         """
-        # Test that `*nums` argument should be Uninferable
         ast = parse(code, __name__)
         *_, add_call, print_call = list(ast.nodes_of_class(nodes.Call))
         nums_arg = add_call.args[0]
         add_call_site = self._call_site_from_call(add_call)
-        self.assertEqual(add_call_site._unpack_args([nums_arg]), [Uninferable])
+        add_call_site_args = add_call_site._unpack_args([nums_arg])
+        self.assertEqual(len(add_call_site_args), 2)
+        self.assertTrue(all(isinstance(arg, nodes.Const) for arg in add_call_site_args))
+        self.assertEqual([arg.value for arg in add_call_site_args], [1, 2])
```

Comment:
This assertion broke and I updated it. It is related to https://github.com/pylint-dev/astroid/pull/2117. From my understanding, `nums` is inferred as `()` and `(1, 2)`. Previously, it was asserted as `Uninferable` because of `safe_infer`. With these changes, the `if nums` guard filters out falsy `()`, and `safe_infer` should now return `(1, 2)`.